src/ui/metric-legend.vue

```diff
@@ -2,27 +2,56 @@
   <li
     :class="'legend_item legend_item_' + position"
   >
-    <div
-      :class="metric.popupKey"
-      :style="{ backgroundColor: metric.color }"
-    />
-    <span class="metricText">&nbsp;{{ metric.htmlName }}</span>
-    <span
-      v-if="metric.description"
-      class="metricText"
-    >&nbsp;-
-      {{ metric.description }}</span>
-    &nbsp;
-    <img
-      class="metricImg metricImgLeft clickable"
-      src="img/icons/pencil.svg"
-      @click="metricPopup"
-    >
-    <img
-      class="metricImg clickable"
-      src="img/icons/trash.svg"
-      @click="trashcanClicked"
-    >
+    <table>
```

Comment:
Just so that we have said it: tables shouldn't be used in modern HTML/CSS to fix styling problems. So, there must be a solution to this problem without a table using copious amounts of JS frameworks, CSS frameworks, goat sacrifices, dark rituals, and swearing like an old sailor.